conda.recipe/build.sh

```diff
@@ -50,9 +45,16 @@ export PARQUET_INSECURE_CURL=1
 source thirdparty/versions.sh
 export GTEST_HOME=`pwd`/thirdparty/$GTEST_BASEDIR
 
+if [ `uname` == Linux ]; then
+    SHARED_LINKER_FLAGS='-static-libstdc++'
```

Comment:
As far as I know, this is really discouraged. e.g. from the GCC Manual:

> There are several situations in which an application should use the shared libgcc instead of the static version. The most common of these is when the application wishes to throw and catch exceptions across different shared libraries. In that case, each of the libraries as well as the application itself should use the shared libgcc.

https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html